activesupport/lib/active_support/core_ext/string/output_safety.rb

```diff
@@ -139,6 +139,10 @@ def to_s
       self
     end
 
+    def as_json
```

Comment:
Isn’t `#as_json` supposed to accept at least one argument? I am currently looking at a stacktrace where this here method is called from https://github.com/rails/rails/blob/e1b0afc155628da4fb950d80f2e6a7bbcdb3e307/activesupport/lib/active_support/core_ext/object/json.rb#L192 with an `options` argument.

Reply:
You are correct.

Reply:
Thank you ❤️

Reply:
Yes it should sorry 🤦